Formula/b/bdo-cli.rb

```diff
@@ -0,0 +1,53 @@
+class BdoCli < Formula
+  desc "Collection of helpful shell scripts for Git and GitHub operations"
+  homepage "https://github.com/hellobdo/bdo-cli"
+  url "https://github.com/hellobdo/bdo-cli/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "5c67911ed56cc628fc88f51d7866b1af0ba3cb452e66ada1bd438ad07ba4c123"
+  license "MIT"
+  
+  depends_on "gh"
+
+  def install
+    # Create scripts directory in the Homebrew prefix
+    (prefix/"scripts").install Dir["scripts/*.sh"]
```

Comment:
You can add this to libexec, but we want to keep the prefix working like any other unix system